resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -547,18 +547,18 @@ cloud_debug.instrument.production_warning.title=Warning!
 lambda.run_configuration.debug_host=Debug Host:
 lambda.sam.debugHost.tooltip=The host to connect the debugger to
 
-feedback.title=Send Quick Feedback...
+feedback.title=Submit Quick Feedback...
```

Comment:
Do We Need Title Case Here? Over-using titlecase makes it meaningless. And it's tacky.

Reply:
Yeah they suggest title case for dialogs/actions: https://jetbrains.github.io/ui/text/capitalization/